main/main.cpp

```diff
@@ -554,6 +555,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	globals = memnew(ProjectSettings);
 
 	register_core_settings(); //here globals are present
+	register_core_extensions(); //before display
```

Comment:
The "before display" comment is not really accurate anymore.

Reply:
I agree. Let me know if my update makes sense.